code/game/objects/structures/girders.dm

```diff
@@ -190,6 +190,10 @@
 				return
 
 		if(!sheets.has_unique_girder && sheets.material_type)
+			if(istype(src, /obj/structure/girder/reinforced))
+				to_chat(user, span_warning("You need plasteel to make a reinforced wall!"))
```

Comment:
This might be better as a balloon

Reply:
The entire proc could use balloons as well as a good refactor but this is just a hotfix. (some of these checks should be in /obj/structure/girder/reinforced)